client/src/css/structure.css

```diff
@@ -881,10 +881,21 @@ a[disabled] {
 }
 
 .expired {
-  /* text-decoration : line-through; */
-  background-color: #ffb3b3 !important;
+  background-color: #ff000083 !important;
+}
+
+.at-risk {
```

Comment:
Yes, it is confusing but lot variables for those are being computed around line https://github.com/IMA-WorldHealth/bhima/blob/master/server/controllers/stock/core.js#L788.   Once we have this function set up, it will be simple to update once we clarify/improve the "risk" calculations.